### PR TITLE
Replaced decomp list with pret.github.io link

### PR DESCRIPTION
**Pull request by AnonymousRandomPerson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,30 +21,7 @@ To set up the repository, see [INSTALL.md](INSTALL.md).
...
-* [**Pokémon Diamond and Pearl**](https://github.com/pret/pokediamond)
-* [**Pokémon Platinum**](https://github.com/pret/pokeplatinum) 
-* [**Pokémon HeartGold and SoulSilver**](https://github.com/pret/pokeheartgold)
-* [**Pokémon Mystery Dungeon: Explorers of Sky**](https://github.com/pret/pmd-sky)
-
-## Contacts
-
-You can find us on:
-
-* [Discord (PRET, #pokecrystal)](https://discord.gg/d5dubZ3)
```
[the remark this answers is not in the document]
Comment (AnonymousRandomPerson) [reply README.md:46]: It's in the site as well, though I've added it back for the extra visibility.

**Comment by mid-kid** [comment]
Why was the IRC link removed?

**Comment by AnonymousRandomPerson** [comment]
Not sure, it can be added back to [pret.github.io](https://github.com/pret/pret.github.io) if that is an issue.

**Comment by mid-kid** [comment]
I would appreciate that, yeah.

**Comment by AnonymousRandomPerson** [comment]
I see there's already a PR (https://github.com/pret/pret.github.io/pull/3). Might need to poke Pikalax about it and/or add more maintainers to the repo.

**Comment by Rangi42** [comment]
Agreed that https://github.com/pret/pret.github.io/pull/3 should merge before this.

**Review by vulcandth — APPROVED** [review]

**Comment by vulcandth** [comment]
> Looks like [pret/pret.github.io#4](https://github.com/pret/pret.github.io/pull/4), which is similar, was already merged. What's left to merge this PR? @vulcandth @Rangi42 @mid-kid

It seems okay to me; if there are no last minute objections i'll merge this soon.

**Pull request merged**